src/master-playlist-controller.js

```diff
@@ -360,6 +360,11 @@ export class MasterPlaylistController extends videojs.EventTarget {
       this.updateDuration();
 
       if (!updatedPlaylist.endList) {
+        // kick segmentloader back into gear if the playlist isn't
```

Comment:
I think it's because we don't want the segment requests to go off too much and VOD triggers the ```loadedplaylist``` immediately. cc @mjneil